airflow/models/__init__.py

```diff
@@ -86,6 +87,8 @@ def __getattr__(name):
 __lazy_imports = {
     "DAG": "airflow.models.dag",
     "ID_LEN": "airflow.models.base",
+    "Backfill": "airflow.models.backfill",
+    "BackfillDagRun": "airflow.models.backfill",
```

Comment:
My 2c, let's not even add these here.

Reply:
IIRC i needed to add them _somewhere_ in the module for it to work.
i'll take a look.

Reply:
I think it's needed in `import_all_models`, not here :)

Reply:
ok done thanks